src/Plugin.php

```diff
@@ -203,6 +203,9 @@ public static function init() {
 
     // Disables output of related products if over-ride checkbox is enabled.
     add_action('woocommerce_after_single_product_summary', __NAMESPACE__ . '\WooCommerce::disableRelatedProducts', 0, 2);
+
+    // Removes the `/page/1` suffix from archive URLs.
```

Comment:
Avoid backticks in PHP code whenever possible, they are the shorthand operator for executing shell commands:
https://www.php.net/manual/en/language.operators.execution.php

```suggestion
    // Removes the suffix '/page/1' from archive URLs.
```